iphone/Maps/UI/PlacePage/Components/PlacePageInfoViewController.swift

```diff
@@ -114,6 +114,9 @@ class PlacePageInfoViewController: UIViewController {
 
   override func viewDidLoad() {
     super.viewDidLoad()
+    setupViews()
```

Comment:
Why is this call necessary? How did it work without it before? Should views be set up before DidLoad?

Reply:
This is small refactoring: I move the setup code into the private method to make viewdidiload clear as it the delegates method.


Views should not be setup before the super.viewdidload - it's important to complete the parent's loading setup.